src/core/ext/xds/xds_api.cc

```diff
@@ -2116,8 +2116,10 @@ grpc_error_handle DownstreamTlsContextParse(
           "TLS configuration provided but no "
           "tls_certificate_certificate_provider_instance found.");
     }
+    return GRPC_ERROR_NONE;
   }
-  return GRPC_ERROR_NONE;
+  return GRPC_ERROR_CREATE_FROM_COPIED_STRING(
```

Comment:
Let's handle the error case right when it comes up instead of wrapping the entire body of the function in an extra level of indentation.  In other words, instead of this:

```
if (value is good) {
  // ...use value...
  return GRPC_ERROR_NONE;
}
return GRPC_ERROR_CREATE_FROM_COPIED_STRING(...);
```

Let's do this:

```
if (value is not good) {
  return GRPC_ERROR_CREATE_FROM_COPIED_STRING(...);
}
// ...use value...
return GRPC_ERROR_NONE;
```